### DISCUSSION TITLE
The PR description says patch update, but matchUpdateType says minor.

### DISCUSSION THREAD

**Discussion by miyajan** [renovatebot/renovate#9095] [Request Help] [answered]
**What Renovate type, platform and version are you using?**

Renovate for GitHub

**Describe the bug**

The PR description by renovate shows it as a patch update, as shown below.

<img width="569" alt="screenshot- 2021-03-12 11 20 27" src="https://user-images.githubusercontent.com/945853/110882262-096b7d00-8325-11eb-9bb4-460eca48d47a.png">

But the `minor` rule of `matchUpdateTypes` is applied instead of `patch`.

For example, I wrote the following config.

```
{
  "extends": [
    "config:base"
  ],
  "packageRules": [
    {
      "matchUpdateTypes": ["patch"],
      "labels": ["UPDATE-PATCH"]
    },
    {
      "matchUpdateTypes": ["minor"],
      "labels": ["UPDATE-MINOR"]
    }
  ]
}
```

The `UPDATE-MINOR` label is added to the PR.

**Relevant debug logs**

The renovate dashboard log shows that the `updateType` was set to `minor`, as shown below.

```
DEBUG: packageFiles with updates
{
  "config": {
    "gomod": [
      {
        "packageFile": "go.mod",
        "constraints": {
          "go": "1.15"
        },
        "deps": [
          {
            "managerData": {
              "lineNumber": 6,
              "multiLine": true
            },
            "depName": "github.com/chromedp/chromedp",
            "depType": "require",
            "currentValue": "v0.6.7",
            "datasource": "go",
            "depIndex": 0,
            "updates": [
              {
                "currentVersion": "v0.6.7",
                "newVersion": "v0.6.8",
                "newValue": "v0.6.8",
                "bucket": "non-major",
                "newMajor": 0,
                "newMinor": 6,
                "updateType": "minor",
                "isSingleVersion": true,
                "branchName": "renovate/github.com-chromedp-chromedp-0.x"
              }
            ],
            "warnings": [],
            "sourceUrl": "https://github.com/chromedp/chromedp",
            "fixedVersion": "v0.6.7"
          }
        ]
      }
    ]
  }
}
```

**Have you created a minimal reproduction repository?**

https://github.com/miyajan/reproduce-renovate-issue/pull/1
This issue is reproduced in this PR.

- [ ] This is a really small bug, it does not need a reproduction (think small typo)
- [x] I have provided a minimal reproduction repository
- [ ] I don't have time for that, but it happens in a public repository I have linked to
- [ ] I don't have time for that, and cannot share my private repository
- [ ] The nature of this bug means it's impossible to reproduce publicly

**Additional context**

- Go modules
- The package version is v0.X.X

**Comment by viceice** [comment, marked as answer]
Renovate doesn't distinguish between minor and patch by default to reduce noise.

See: https://docs.renovatebot.com/configuration-options/#separateminorpatch

**Reply by miyajan** [reply]
Thank you for your help!

**Reply by miyajan** [reply]
But it seems too difficult to realize that I have wrong settings...

**Reply by viceice** [reply]
It's also difficult for use to make a good preset for almost all users.
